libs/clients/fishing-licence/src/lib/SkipApiProvider.ts

```diff
@@ -0,0 +1,23 @@
+import { Provider } from '@nestjs/common'
+import { SkipApi } from './gen/fetch'
+import {
+  ConfigType,
+  IdsClientConfig,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+import { FishingLicenseClientConfig } from './FishingLicenseClientConfig'
+import { FishingLicenseApiFactoryConfig } from './FishingLicenseApiFactoryConfig'
+
+export const SkipApiProvider: Provider<SkipApi> = {
+  provide: SkipApi,
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xRoadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof FishingLicenseClientConfig>,
+    idsConfig: ConfigType<typeof IdsClientConfig>,
+  ) =>
+    new SkipApi(FishingLicenseApiFactoryConfig(xRoadConfig, config, idsConfig)),
```

Comment:
I guess you saw the other way to register multiple APIs in the NationalRegistryV2 client.

Reply:
I did not, but now that I have taken a look I wish I had thought of that 😄 I'll change into doing it same way as NationalRegistryV2, feels cleaner